test/sim/abilities/parentalbond.js

```diff
@@ -4,65 +4,68 @@ const assert = require('./../../assert');
 const common = require('./../../common');
 
 let battle;
-let basePowers;
+let damages;
 
 describe('Parental Bond', function () {
 	beforeEach(function () {
 		battle = common.createBattle([
-			[{species: 'Kangaskhan', ability: 'parentalbond', item: 'normaliumz', moves: ['falseswipe', 'doublehit']}],
-			[{species: 'Aggron', ability: 'noguard', moves: ['rest']}],
+			[{species: 'Kangaskhan', ability: 'parentalbond', item: 'electriumz', moves: ['thunderpunch', 'doublekick']}],
+			[{species: 'Aggron', ability: 'battlearmor', moves: ['rest']}],
 		]);
 
-		basePowers = [];
-		battle.onEvent('BasePower', battle.format, -9, function (bp, attacker, defender, move) {
-			basePowers.push(this.modify(bp, this.event.modifier));
+		damages = [];
+		battle.onEvent('ModifyDamage', battle.format, -9, function (damage, attacker, defender, move) {
+			damages.push(damage);
 		});
 	});
 
 	afterEach(function () {
 		battle.destroy();
 	});
 
-	it('should cause single-hit attacks to strike twice, with the second hit at 0.25 power', function () {
-		battle.makeChoices('move falseswipe', 'move rest');
-		assert.deepEqual(basePowers, [40, 10]);
+	it('should cause single-hit attacks to strike twice, with the second hit dealing 0.25x damage', function () {
+		battle.makeChoices('move thunderpunch', 'move rest');
+		const damageRatio = damages[0] / damages[1];
+		assert(damageRatio > (0.85 / 0.25 - 0.1) && damageRatio < (1 / (0.85 * 0.25) + 0.1), `Damage ratio ${damageRatio} not within expected range`);
 	});
 
 	it('should not have any effect on moves with multiple hits', function () {
-		battle.makeChoices('move doublehit', 'move rest');
-		assert.deepEqual(basePowers, [35, 35]);
+		battle.makeChoices('move doublekick', 'move rest');
+		assert(damages[0] >= 54 && damages[0] <= 64 && damages[1] >= 54 && damages[1] <= 64, damages);
 	});
 
 	it('should not have any effect Z-Moves', function () {
-		battle.makeChoices('move falseswipe zmove', 'move rest');
-		assert.deepEqual(basePowers, [100]);
+		battle.makeChoices('move thunderpunch zmove', 'move rest');
+		assert.equal(damages.length, 1);
+		assert(damages[0] >= 61 && damages[0] <= 72);
```

Comment:
I believe these can be replaced with assert.Bounded(damages[0], [61, 72]); and in similar ways elsewhere